src/components/templates/PosterPage/components/ShareModal/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { FACEBOOK_APP_ID } from "secrets";
```

Comment:
[NB] We tend to call these files `utils` rather than `helpers`.

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/ce99953a2c8fa07fc9ad3110344481163911e470